playbook/app/pb_kits/playbook/pb_advanced_table/_advanced_table.scss

```diff
@@ -94,6 +94,10 @@
     color: $primary !important;
   }
 
+  .bg-silver {
```

Comment:
I think we already have this classname in this scss file and we don't need to re-add it here!